integration-tests/rest-tests-jaxrs/src/test/groovy/org/hawkular/metrics/alerting/ExternalAlerterITest.groovy

```diff
@@ -61,10 +68,12 @@ class ExternalAlerterITest extends AlertingITestBase {
         resp = client.post(path: "triggers", body: triggerTestAvg)
         assertEquals(200, resp.status)
 
-        // ADD external metrics avg condition. Note: alerterId must be "HawkularMetrics"
+        // Note: alerterId must be "HawkularMetrics"
         // Average over the last 1 minute > 50, check every minute
+        Query qNow = Query.builder("qNow").metric(metricId).duration("1mn").build();
```

Comment:
I would also add tests over the REST API through plain JSON too.
Perhaps the same but with plain text.
There are not support for extensions with Java clients, and also, I am not 100% if python/ruby clients are tested against ExternalCondition (probably not).
So, these JSON tests will be the closest example to a live example.
It would help to point people into the feature.